crop_classification/inference.py

```diff
@@ -0,0 +1,116 @@
+from cropharvest.engineer import Engineer
+from datetime import datetime
+from pathlib import Path
+import numpy as np
+import xarray as xr
+import pandas as pd
+import re
+import torch
+
+from typing import Dict, List, Optional, Tuple, Union
+
+
+class Inference:
+    """
+    Class for running inference using either a pytorch checkpoint model or
+    a pytorch jit model on a single tif file.
+    """
```

Comment:
@gabrieltseng would adding more detail here (or to the top of the file) be more useful than the readme? It would be more tightly coupled to this file

Reply:
yes, that works too. I think a brief thing in the readme would be good as well though (could literally be a single line)